roles/edpm_derive_pci_device_spec/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Derive the SR-IOV device_spec config map for nic-partitionin
+  ansible.builtin.import_tasks: derive_pci.yml
```

Comment:
@vcandapp please change derive_pci.yml file name like derive_pci_device_spec.yml.

Reply:
Changed file name for more clarity